websocket_server.cpp

```diff
@@ -27,21 +28,31 @@ WebSocketServer::WebSocketServer(QObject *parent, quint16 port) :
         ports[0] = port;
     }
 
-    while (!srv->listen(QHostAddress::AnyIPv4, ports[p]))
+    QHostAddress address;
+    if (deCONZ::appArgumentString("--http-listen", 0).isEmpty()) // Tie websocket server to specified IP, if any
```

Comment:
Almost there, I've noticed the second default argument should be a string instead 0.

So in this case:

`deCONZ::appArgumentString("--http-listen", QString())`